src/es_sfgtools/processing/operations/sv3_ops.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+from pydantic import BaseModel, Field, ValidationError
+from datetime import datetime, timedelta
+from typing import Optional
+import os
+import logging
+import json
+import pandera as pa
+from pandera.typing import DataFrame
+import pymap3d as pm
+from warnings import warn
+from ..assets.observables import ShotDataFrame
+from ..assets.file_schemas import DFPO00RawFile, QCPinFile
+from ..assets.constants import TRIGGER_DELAY_SV3
+from ..assets.logmodels import SV3InterrogationData,SV3ReplyData,get_traveltime,get_triggertime,check_sequence_overlap
+logger = logging.getLogger(os.path.basename(__file__))
+
+
+
+
+
+@pa.check_types
+def check_df(df: pd.DataFrame) -> pd.DataFrame:
+    df = check_sequence_overlap(df)
+    return df
+
+
+def dev_dfop00_to_shotdata(source: DFPO00RawFile) -> DataFrame[ShotDataFrame]:
+
+    processed = []
+    interrogation = None
+    with open(source.local_path) as f:
+        lines = f.readlines()
+        for line in lines:
+            data = json.loads(line)
+            if data.get("event") == "interrogation":
```

Comment:
might be good to include a try/except here, I've seen .get() fail if theres fields missing from a file, and we dont want a single bad event to prevent us from getting the whole file